src/main/kotlin/com/dh/ondot/schedule/infra/api/EverytimeApi.kt

```
@@ -0,0 +1,105 @@
+                .contentType(MediaType.APPLICATION_FORM_URLENCODED)
+                .body("identifier=${URLEncoder.encode(identifier, StandardCharsets.UTF_8)}&friendInfo=true")
+                .retrieve()
+                .body(String::class.java)
+
+            if (response.isNullOrBlank()) {
+                throw EverytimeNotFoundException()
+            }
+            return response
+        } catch (ex: EverytimeNotFoundException) {
+            throw ex
+        } catch (ex: RestClientResponseException) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)